robottelo/helpers.py

```diff
@@ -521,7 +521,8 @@ def host_provisioning_check(ip_addr):
     :return: ssh command return code and stdout
     """
     result = ssh.command(
-        f'for i in {{1..60}}; do ping -c1 {ip_addr} && exit 0; sleep 20; done; exit 1'
+        f'for i in {{1..60}}; do ping -c1 {ip_addr} && exit 0; sleep 20; done; exit 1',
+        timeout = 2000 # cycle itself should take at most 1800s (ping has implicit timeout 10)
```

Comment:
Consider using `-W 2` on the ping command to reduce the timeout for each  attempt.

Reply:
Reducing the timeout isn't something I want to do. Provisioning an OS on the VM takes some time, I can't expect it to be less than the default 10s. More like half an hour.

Reply:
Not reducing the overall timeout. Reducing the amount of time ping sits waiting for a response (-W), or the sleep between sending pings.

Reply:
That will just make this whole thing more prone to network caused failures, without any benefit I can see.